test/library/standard/CommDiagnostics/commDiagnosticsWarnings.prediff

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# get rid of the 'executeOn' lines from verboseComm
+cat $2 | grep -v 'executeOn' > $2.prediff.tmp
```

Comment:
I wonder if you should just grep for `warning` here. Some unforeseen changes could make this test generate comm other than `on`, in which case this will not be sufficient. Just checking for `warning` could make this test more robust IMO.